pkg/instrumentation/annotation.go

```diff
@@ -31,6 +31,21 @@ const (
 	annotationInjectApacheHttpdContainersName = "instrumentation.opentelemetry.io/apache-httpd-container-names"
 	annotationInjectNginx                     = "instrumentation.opentelemetry.io/inject-nginx"
 	annotationInjectNginxContainersName       = "instrumentation.opentelemetry.io/inject-nginx-container-names"
+	annotationJMXJVM                          = "cloudwatch.aws.amazon.com/inject-jmx-jvm"
+	annotationJMXTomcat                       = "cloudwatch.aws.amazon.com/inject-jmx-tomcat"
+	annotationJMXKafka                        = "cloudwatch.aws.amazon.com/inject-jmx-kafka"
+	annotationJMXKafkaConsumer                = "cloudwatch.aws.amazon.com/inject-jmx-kafka-consumer"
+	annotationJMXKafkaProducer                = "cloudwatch.aws.amazon.com/inject-jmx-kafka-producer"
```

Comment:
Consider keeping these in their own `const` block.

Reply:
Or even in a separate file entirely.

Reply:
I'll make it another const block. I disagree with diff file, since all annotations should be here.